.gdbinit

```diff
@@ -1,6 +1,8 @@
+set tcp connect-timeout 30
 target remote localhost:1234
 break main
 continue
 
 source debug/ktrace.py
 source debug/kdump.py
+source debug/mutex.py
```

Comment:
I realised it is better not to do force pushing because this messes up github history. Since these changes work, are tested by me, reviewed, and obviously associated with mutexes it is OK I guess to put them in one pull request.

I think it is good idea to add GDB scripts to every new subsystem associated PR. So this is something like 'system + scripts' or nothing, because in case something happens we might have easier time debugging it, because we instantly get the scripts.